RELEASING.md

```diff
@@ -34,3 +34,34 @@ Repositories create consistent release labels, such as `v1.0.0`, `v1.1.0` and `v
 The release process is standard across repositories in this org and is run by a release manager volunteering from amongst [MAINTAINERS](MAINTAINERS.md).
 
 TODO
+
+## Backporting Guide
+
+Here are the commands we typically run to backport changes to release branches.
+
+* First, checkout the target release branch and pull the latest changes from `upstream`. In the examples below, our target release branch is `1.x`
```

Comment:
I would remove First, Next, Finally, because you're missing a word for the 3rd item :)